pkg/chunk/schema_config.go

```diff
@@ -210,6 +214,17 @@ func (cfg PeriodConfig) validate() error {
 		return errInvalidTablePeriod
 	}
 
+	switch cfg.Schema {
+	case "v1", "v2", "v3", "v4", "v5", "v6", "v9":
```

Comment:
Even if I guess it doesn't cause any harm, to avoid any misunderstanding/misconfiguration from the user side, shouldn't we ensure the `cfg.RowShards` for these schemas is `0`? Am I missing anything?